src/sass/variables/_Font.Variables.scss

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
+
+// Font name
+$ms-font-name: 'Segoe UI' !default;
+
+// Font directory and paths
+$ms-font-directory:         'https://static2.sharepointonline.com/files/fabric/assets/fonts/' !default;
+$ms-font-path-arabic:       'segoeui-arabic' !default;
+$ms-font-path-cyrillic:     'segoeui-cyrillic' !default;
+$ms-font-path-easteuropean: 'segoeui-easteuropean' !default;
+$ms-font-path-greek:        'segoeui-greek' !default;
+$ms-font-path-hebrew:       'segoeui-hebrew' !default;
+$ms-font-path-vietnamese:   'segoeui-vietnamese' !default;
+$ms-font-path-westeuropean: 'segoeui-westeuropean' !default;
+
+// Base font stacks
+$ms-font-system-base: 'Segoe UI', -apple-system, BlinkMacSystemFont, 'Roboto', 'Helvetica Neue', sans-serif !default;
+$ms-font-family-base: 'Segoe UI WestEuropean', $ms-font-system-base !default;
```

Comment:
The web font is spelled as 'WestEuropean': https://github.com/OfficeDev/office-ui-fabric-core/blob/master/src/sass/mixins/_Font.Mixins.scss#L141

I'm going to open a new issue to include "Web Font" in the name of any `@font-face` we declare. It's difficult to tell right now where we're expecting a system font to load and where it's a web font we've defined, which we can name whatever we like. 😄